charts/observability-pipelines-worker/README.md

```diff
@@ -20,15 +20,15 @@ To install the chart with the release name `<RELEASE_NAME>` run:
 ```bash
 helm install --name <RELEASE_NAME> \
   --set datadog.apiKey=<DD_API_KEY> \
-  --set datadog.configKey=<DD_OP_CONFIG_KEY> \
+  --set datadog.pipelineID=<DD_OP_PIPELINE_ID> \
   datadog/observability-pipelines-worker
 ```
 
-By default, this chart creates secrets for your Observability Pipelines API and configuration keys. However, you can use
-manually created Secrets by setting the `datadog.apiKeyExistingSecret` and/or `datadog.appKeyExistingSecret` values
+By default, this chart creates secrets for your Observability Pipelines API key and pipeline ID. However, you can use
+manually created Secrets by setting the `datadog.apiKeyExistingSecret` and/or `datadog.pipelineIDExistingSecret` values
```

Comment:
```suggestion
manually created Secrets by setting the `datadog.apiKeyExistingSecret` and/or `datadog.pipelineIdExistingSecret` values
```

If we're being super proper/pedantic it should be `Id`.

Reply:
😢 Alas, I flip-flopped on this and landed on ID. Let's go for super proper 👍

Reply:
I'll bundle the renaming's into a single commit.

Reply:
Adjusted in [7a5e07c](https://github.com/DataDog/helm-charts/pull/952/commits/7a5e07c5b852e9393a9f49a75395fe08597c52dd)